docs/themes/thxvscode/layouts/apis/list.html

```diff
@@ -30,7 +30,10 @@ <h2>{{ $grouping | humanize | title }} packages</h2>
                         {{ range where $packagePages "Params.package" . }}
                         <tr>
                             <td><a href="{{ .RelPermalink | safeURL }}" title="{{ .Title }}">{{ .Title }}</a></td>
-                            <td>{{ truncate 170 (.Params.Summary | markdownify) }}</td>
+                            <td>
+                                {{ $firstLine := index (split .Params.Summary "\n\n") 0 }}
+                                {{ $firstLine | markdownify }}
```

Comment:
```suggestion
                                {{ $firstLine := index (split .Params.Summary "\n\n") 0 }}
                                {{- $firstLine | markdownify -}}
```

You may need to add the dashes to eat whitespace before/after the template after its rendered. Usually it doesn't matter, but it might with content in `td`s -- not sure. If you tested the rendering and it looks good then ignore this.

Reply:
Good idea!